frontend/src/pages/UnverifiedEmail.vue

```diff
@@ -1,18 +1,25 @@
 <template>
     <ff-layout-box>
-        <form class="px-4 sm:px-6 lg:px-8 mt-8 space-y-6 max-w-md" @submit.prevent>
+        <form class="px-4 sm:px-6 lg:px-8 mt-8 mx-auto space-y-6 max-w-md" @submit.prevent>
             <p>
                 Before you can access the platform, we need to verify your email
                 address.
             </p>
             <p>
-                We sent you an email with a link to click when you signed up.
+                We sent you an email with a code in it. Enter the code below to continue.
+            </p>
+            <div>
+                <ff-text-input v-model="token" data-form="verify-token" maxlength="6" label="token" @enter="submitVerificationToken" />
```

Comment:
Ideally this would have `autocomplete="one-time-code"` ([reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#values)) to prevent suggesting passwords and such

without correct autofill we get this: 
![image](https://github.com/user-attachments/assets/2e1500e2-0301-445e-86b0-94b866041f5c)

Reply:
Noted - I'd like to improve the style of the OTC input as we have it in multiple places now. Will include this suggestion in that, rather than diverge.